mainapp/helpers.py

```diff
@@ -186,10 +187,34 @@ def combine_ids(cv_bookids, embedding_bookids, already_rated, recommendations=9)
 
     if len(best_bookids) < recommendations:
         # If not enough recommendations
-        best_bookids = best_bookids + cv_bookids[3:(recommendations + 3 - len(best_bookids))]
+        two_n = (recommendations - len(best_bookids))
+        n1, n2 = math.ceil(two_n/2), math.floor(two_n/2)
+        best_bookids_remain = cv_bookids[3: 6+n1]
```

Comment:
If you put `6` because of duplicates, Specify `3 * 2`, for more clarity, +1 if you add comments wherever possible, since for a new contributor it's a bit difficult to catch up